inference-engine/thirdparty/clDNN/kernel_selector/common/common_types.h

```diff
@@ -7,6 +7,10 @@
 #include <cstddef>
 #include <cstdint>
 
+#define CLDNN_THREADING_SEQ 0
+#define CLDNN_THREADING_TBB 1
+#define CLDNN_THREADING_THREADPOOL 2
```

Comment:
Why these definitions are located in kernel selector header while kernel selector doesn't use threading?

Reply:
they are defined in the engine.hpp in the latest master, so I think we can move this to there.

Reply:
offline discuss => we'll define them in custom_arena.h

Reply:
Moved these defined to the custom_task_arean.h. review it again. thanks.